tests/chart_tests/test_astronomer_houston_hook_job.py

```diff
@@ -94,11 +105,44 @@ def test_db_migration_job_defaults(self, kube_version):
             "runAsNonRoot": True
         }
 
-        assert c_by_name["houston-db-migrations-job"]["args"] == [
-            "yarn",
-            "migrate",
-        ]
+        assert c_by_name["houston-db-migrations-job"]["args"] == ["yarn", "migrate"]
 
         assert c_by_name["houston-db-migrations-job"]["securityContext"] == {
             "runAsNonRoot": True
         }
+        assert "resources" in c_by_name["wait-for-db"]
+        assert "resources" in c_by_name["houston-bootstrapper"]
+        assert "resources" in c_by_name["houston-db-migrations-job"]
+
+    def test_db_migration_job_custom_resources(self, kube_version):
+        """Test Db Migration Job with customer resources."""
+
+        overridden_resources = {
+            "requests": {"cpu": "300m", "memory": "300Mi"},
+            "limits": {"cpu": "700m", "memory": "700Mi"},
+        }
+
+        value = {"astronomer": {"houston": {"resources": overridden_resources}}}
+
+        docs_overridden = render_chart(
+            kube_version=kube_version,
+            values=value,
+            show_only=[
+                "charts/astronomer/templates/houston/helm-hooks/houston-db-migration-job.yaml"
+            ],
+        )
+        assert len(docs_overridden) == 1
+        c_by_name_overridden = get_containers_by_name(
+            docs_overridden[0], include_init_containers=True
+        )
+
+        assert c_by_name_overridden["wait-for-db"]["resources"] == overridden_resources
+        assert (
+            c_by_name_overridden["houston-bootstrapper"]["resources"]
+            == overridden_resources
+        )
+        assert (
+            c_by_name_overridden["houston-db-migrations-job"]["resources"]
+            == overridden_resources
+        )
+#EOF
```

Comment:
What is the purpose of this?

Reply:
EOF is added due to one of the checks that is failing:  fix end of files
So i was trying this out with a comment and a newline.

Reply:
fix-end-of-files makes sure that all whitespace is trimmed off the end of the file, with only one trailing newline. If you `pre-commit install` and then `pre-commit run --all-files` in your local checkout then it'll fix this all locally for you. The idea here is that these changes will be made when you `git commit` if anything is out of line, you can review the updates, `git add --update` and then `git commit` again.